content/docs/sync-async-boundary.md

```diff
@@ -0,0 +1,43 @@
+# Sync-async boundary
+
+If my experience in the IT industry has taught me anything it would be that drawing boundaries is the most important part of the design process. Boundaries are essential for understanding and communicating the design. The sync-async boundary is an example of af a boundary that is useful when designing distributed systems.
```

Comment:
``` suggestion
If my experience in the IT industry has taught me anything it would be that drawing boundaries is the most important part of the design process. Boundaries are essential for understanding and communicating the design. The sync-async is an example of a boundary that is useful when designing distributed systems.